components/brave_new_tab_ui/components/default/braveToday/cardsGroup.tsx

```diff
@@ -17,7 +17,6 @@ import CardType = BraveNews.CardType
 
 // Disabled rules because we have a function
 // which returns elements in a switch.
-// tslint:disable:jsx-wrap-multiline jsx-alignment
```

Comment:
Maybe we need to remember to enable jsx alignment in either prettier or eslint

Reply:
I believe that would be part of eslint-react-plugin and not prettier? This would require us to add another package in the future.

Reply:
Let's create / add to the existing issue to add that when looking at formatting options

Reply:
https://github.com/brave/brave-browser/issues/19405